bin/compile

```diff
@@ -44,7 +44,8 @@ if test -n "$meteor_version" ; then
   # Most Meteor projects are using React or equivalent and all these
   # dependencies are in dev usually. We had way too much support because of
   # this.
-  export NPM_CONFIG_PRODUCTION=false
+  npm_config_production=${NPM_CONFIG_PRODUCTION:-false}
```

Comment:
Wont this be already set to true by the `create_default_env` method of `lib/environment.sh` ?

Reply:
`create_default_env` is called on line 96 of this file, so it will be called later. This won't be overriden.